kubernetes/apps/overlays/gtfs-rt-archiver-v3-prod/archiver-channel-vars.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: archiver-channel-vars
+data:
+  AIRFLOW_ENV: development
```

Comment:
Is this the correct value for the prod overlay?

Reply:
Ah yeah prior to now because I wanted to write to the test bucket while I was tinkering with file keys. Will fix.

Reply:
Also like `AIRFLOW_ENV` is clearly a sign of code smell in calitp-py (that I introduced).